intake/interface/source/defined_plots.py

```diff
@@ -12,6 +12,12 @@
 except ImportError:
     dfviz = False
 
+try:
+    import hvplot
+    assert LooseVersion(hvplot.__version__) >= LooseVersion("0.8.1")
```

Comment:
`LooseVersion` is deprecated, no? Suggest `packaging.Version`.

Is there an argument for requiring hvplot?

Reply:
Changing to `packaging.version.Version`.

Reply:
I don't know if requiring hvplot makes sense for intake as a whole, but I could see requiring hvplot when someone accesses `intake.gui`. I'll add that so that the GUI experience is better for users.